### PR TITLE
Add Dashboard project info

### PR DESCRIPTION
**Pull request by flaviocastro**
This PR adds the Dashboard Project Info card with all the data from the Blockchain.

| Desktop | Tablet | Mobile | 
| ---- | ---- | ---- |
| image |  image | image |

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -57,6 +59,7 @@
     "@typescript-eslint/parser": "^5.7.0",
     "@untile/eslint-config-untile-react": "^1.0.0",
     "@untile/stylelint-config-untile": "^1.0.0",
+    "copy-webpack-plugin": "^10.2.4",
```
Comment (rafaelcruzazevedo) [inline packages/web/package.json:62]: It is no longer needed

File: packages/web/src/core/utils/formatters.ts [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+/**
+ * Export `formatCompactNumber`.
+ */
+
+export function formatCompactNumber(
+  value: NullableNumber,
+  options?: CurrencyOptions
+): string {
+  const { currency, decimalPlacesToDisplay = 1 } = options ?? {};
+
```
Comment (rafaelcruzazevedo) [inline packages/web/src/core/utils/formatters.ts:192]: remove?

File: packages/web/src/core/utils/formatters.ts [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    compactDisplay: 'short',
+    currency,
+    maximumSignificantDigits,
+    notation: 'compact',
+    style: currency ? 'currency' : undefined
+  }).format(truncatedValue);
+}
+
+/**
+ * `formatDate` connector.
```
Comment (rafaelcruzazevedo) [inline packages/web/src/core/utils/formatters.ts:215]: wrong doc bloc

File: packages/web/package.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "lint:styles": "stylelint 'src/**/*.@(ts|tsx|js)'",
     "lint:types": "tsc --noEmit",
     "precommit": "lint-staged",
+    "prepare:contracts": "cp -r ../contracts/deployments ./contracts",
```
Comment (rafaelcruzazevedo) [inline packages/web/package.json:13]: it's supposed to be manual for now right? because is not called anywhere

**Review by rafaelcruzazevedo — CHANGES_REQUESTED** [review]

File: packages/web/package.json (hunk 4 above)
Comment (flaviocastro) [reply packages/web/package.json:13]: Yes, we will deal with that later.

**Review by rafaelcruzazevedo — APPROVED** [review]

**Pull request merged**